interactive_engine/executor/ir/runtime/src/process/operator/map/get_v.rs

```diff
@@ -159,17 +159,22 @@ impl FilterMapFunction<Record, Record> for AuxiliaOperator {
             // e.g., for g.V().out().as("a").has("name", "marko"), we should compile as:
             // g.V().out().auxilia(as("a"))... where we give alias in auxilia,
             //     then we set tag=None and alias="a" in auxilia
-            // 1. filter by labels.
+
+            // 1. If to filter by labels, and the entry itself carries label information already, directly eval it without query the store
```

Comment:
Can the following code be replaced with params.ha_labels()?

Reply:
Done.